wallet.go

```diff
@@ -1179,6 +1323,22 @@ func (w *Wallet) ResendUnminedTxs() {
 	}
 }
 
+// SortedActiveAccountAddresses returns a slice of addresses of an account in a wallet.
+func (w *Wallet) SortedActiveAccountAddresses(account uint32) ([]string, error) {
```

Comment:
I would just remove this.  There's no reason to need addresses to be sorted.  I think this was previously added to mimic an implementation detail in bitcoind (a `std::map` is usually implemented as an RB tree, so iterating over k/v pairs has a defined order).  Instead, I would change `GetAddressesByAccount` to return a JSON array of all addresses (even those not marked active) in any order.